src/measure_theory/l1_space.lean

```diff
@@ -46,77 +136,101 @@ assume hfi hgi,
 
 -- We don't need `f` to be measurable here, but it's easier to have a uniform API
 @[nolint]
-lemma lintegral_nnnorm_neg {f : α → γ} (hf : measurable f) :
+lemma lintegral_nnnorm_neg {f : α → β} (hf : measurable f) :
```

Comment:
I don't see where you need the measurability of `f` here. Have you tried to put `#lint` at the end of your file? It will tell you about unnecessary assumptions (plus a bunch of other stuff).

Reply:
Sorry, I just saw the comment before the lemma. When the assumption is not needed, it is really better to remove it (I promise it will save you time in the future, when you will have less stuff to check).